x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -770,7 +770,10 @@ public static RoleDescriptor kibanaSystemRoleDescriptor(String name) {
                     .privileges("auto_configure", "read", "write")
                     .build(),
                 // Osquery manager specific action responses. Kibana reads from these to display responses to the user.
-                RoleDescriptor.IndicesPrivileges.builder().indices(".logs-osquery_manager.action.responses-*").privileges("read").build(),
+                RoleDescriptor.IndicesPrivileges.builder()
```

Comment:
__nit:__ would make sense to update the comment as well to reflect new privileges and what are they used for.